code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -715,6 +715,7 @@
 	armor = list("melee" = 50, "bullet" = 30, "laser" = 30, "energy" = 30, "bomb" = 80, "bio" = 0, "rad" = 0, "fire" = 90, "acid" = 90)
 
 /obj/machinery/porta_turret/syndicate/shuttle/target(atom/movable/target)
+	..()
```

Comment:
```suggestion
```
Will cause the parent proc to be called, which runs popUp() (Code that will probably break syndicate turrets), setDir() and shootAt() which will at the very least cause the turrets to fire 2 shots simultaneously on the first burst, and at the worst break the icon state of the turrets.

Reply:
calling the parent fixed an issue I had in testing where the turrets wouldn't fire if they were turned off and on again, the ```has_cover``` variable syndicate turrets is set to false. Doesn't seem to be any runtimes on my end

Reply:
still pretty certain that this will cause the first bullet to deal double damage (2 bullets stacked into one).

Following the code all the way down, replacing every single ..() with what is called in their parent procs ends with this:

```
/obj/machinery/porta_turret/syndicate/shuttle/target(atom/movable/target)
	if(target)
		popUp()				//pop the turret up if it's not already up.
		setDir(get_dir(base, target))//even if you can't shoot, follow the target
		shootAt(target)
	if(target)
		setDir(get_dir(base, target))//even if you can't shoot, follow the target
		shootAt(target)
		addtimer(CALLBACK(src, .proc/shootAt, target), 5)
		addtimer(CALLBACK(src, .proc/shootAt, target), 10)
		addtimer(CALLBACK(src, .proc/shootAt, target), 15)
		return TRUE
```

Reply:
Ah, popup() is what allows the turret to fire again, should work if you replace ..() with pop_up

Reply:
(Or just set raised = TRUE) because otherwise the turret will have a 5 second period in which it cannot shoot

Reply:
Alright after looking at the code and understanding some more

setState(False) the bit that turns it off calls popDown() which disables the turret.
Calling ..() Calls popup, which reanbles the turret however this has a 5 second delay to account for the popup animation which the penetrator turret has.
The best solution I can see is to either otherride the syndicate popup and popdown proc to not do anything, or override setState to not call popDown when turned off.

Reply:
![image](https://user-images.githubusercontent.com/26465327/106482807-c3113b80-64a5-11eb-9e5e-cf57742fadf5.png)

Reply:
is what it should look like